plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -120,6 +120,14 @@ public void testReadFromView()
         }
     }
 
+    @Test
+    public void testSynonyms()
```

Comment:
Let's add a new test class enabling `sqlserver.synonyms.enabled` config property and verify that the connector can access synonyms.